.changeset/wise-badgers-cover.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/types': minor
+---
+
+Introduce `afterLogoClickUrl` prop in `ClerkProvider`
```

Comment:
👋🏻 
What is the use case for this prop? Can we add more info in the description, please as this prop sounds a bit too specific to be in ClerkProvider

Reply:
Is this only going to be used by Hosted Pages for now?

Reply:
no @nikosdouvlis 
is someone use clerk-js will able to pass `afterLogoClickUrl` in order to change the `ApplicationLogo` url or otherwise if someone use hosted pages will able to change it throw dashboard

Reply:
I can also see that if someone has this set in the dashboard, it will also affect the components as we are falling back on it if a prop is not passed to `<ClerkProvider/>`. That's what I mentioned above.

Reply:
I though we were going to ignore this in `clerk-js` and just use the prop on accounts.